package.json

```diff
@@ -69,14 +72,16 @@
     "eslint-plugin-react-hooks": "4.2.0",
     "eslint-plugin-standard": "5.0.0",
     "husky": "6.0.0",
-    "jest": "27.0.6",
+    "jest": "29.3.1",
     "metro-react-native-babel-preset": "^0.73.3",
     "npm-watch": "^0.11.0",
     "prettier": "2.3.2",
     "react": "18.0.0",
     "react-native": "0.69.6",
     "react-native-builder-bob": "0.18.1",
     "react-test-renderer": "18.0.0",
+    "ts-jest": "^29.0.3",
```

Comment:
Are `ts-jest` and `ts-node` needed since `.tsx` files are transpiled by `@swc/jest`?

Reply:
No, they aren't.
The ts-jest is used for `jest.config.ts`. If you install it, you can write jest config files in typescript like `jest-setup.ts`, `jest-environement.ts` so on, and specify ts files in the jest.config file. ts-node is set peerDependency in ts-jest.`@swc/jest` itself can run without ts-jest and ts-node.